tests/contrib/django/test_django_snapshots.py

```diff
@@ -9,11 +10,11 @@
 from tests.webclient import Client
 
 
-SERVER_PORT = 8000
+SERVER_PORT = "8000"
```

Comment:
why is this now a string?

Reply:
It only used as a string. So it makes sense to just remove the str() conversion on line 36 and just use it as a string